app/features/marketplace-form/marketplace-form.tsx

```diff
@@ -91,17 +91,30 @@ export function MarketplaceForm({ projects, tokens }: { projects: Project[]; tok
         ) : null}
       </section>
 
+      <section>
+        <h4 className="font-semibold mb-4">Challenge Rewards</h4>
+        <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+          <Field>
+            <Label>Reward Token Allowlist</Label>

```

Comment:
do we need this since there is no input the contract for a token allow list?

Reply:
I don't know, it's in the designs. I assume not needed with only one token (mvp) and needed once more than one?

Reply:
yeah maybe hide for now